ethereum/contracts/BeefyLightClient.sol

```diff
@@ -528,15 +540,14 @@ contract BeefyLightClient {
         pure
         returns (bytes memory)
     {
-        bytes memory scaleEncodedMMRLeaf =
```

Comment:
ditto. Lets rather agree on & perform formatting changes in a separate PR.

Reply:
this file has been changed all over anyway, so it's not feasible nor important to worry about whitespace mismatch for it i think.